### ISSUE TITLE
Drash.Loggers.ConsoleLogger is not a constructor

### ISSUE THREAD

**Issue by ebebbington** [drashland/deno-drash#136]
**Bug Description**
When creating a 'morgan style' logger, the documentation show we must use the following: `const logger = new Drash.Loggers.ConsoleLogger({...})`. Yet, when compiled, an error is thrown:

`error: Uncaught TypeError: Drash.Loggers.ConsoleLogger is not a constructor`

**To Reproduce**

1. Create a file named `logging_middleware.ts` and paste in the following:
```
import Drash from "https://deno.land/x/drash/mod.ts";

const logger = new Drash.Loggers.ConsoleLogger({
  enabled: true,
  level: "all",
  tag_string: "{datetime} | {level} |",
  tag_string_fns: {
    datetime() {
      return new Date().toISOString().replace("T", " ");
    }
  }
});
```
2. Specify this middleware inside the app file for it to be called:
```
const server = new Drash.Http.Server({
  address: "localhost:1337",
  **middleware: {
    server_level: {
      before_request: [
        LoggingMiddleware
      ],
      after_request: [
        LoggingMiddleware
      ]
    }
  },**
  directory: '.',
  response_output: 'text/html'
});
```
3. Run the application:
`deno --allow-net app.ts`

**Expected behavior**
A logger can be instantiated without any errors

**Suggested Solution(s)**
Maybe the documentation for this is wrong? The section for it is [here](https://drash.land/#/tutorials/middleware/morgan-style-logging-middleware)

**Desktop**

- OS: [Windows 10]
- Browser [N/A]
- Version [N/A]

**Additional context**
Like i said above, i reckon this is down to two things:

1. Documentation for this section is slightly incorrect
2. Logger cannot actually be created in the way the documentation outlines

**Comment by ebebbington** [comment]
Found the issue.

In all documentation references, the usage of logging is: `Drash.Loggers.SomeLogger`. Although, in https://deno.land/x/drash/mod.ts, I see that the namespace Drash exports the Loggers under the namespace "CoreLoggers".

So I edited my  logger implementing to make this change, this is what my `app.ts` file looks like now:
```
import { Drash } from "./deps.ts";
const server = new Drash.Http.Server({
  address: "localhost:1337",
  logger: new Drash.CoreLoggers.ConsoleLogger({ // this line here, note the change from "Loggers." to "CoreLoggers."
    enabled: true,
    level: "all",
    tag_string: "{datetime} | {level} |",
    tag_string_fns: {
      datetime() {
        return new Date().toISOString().replace("T", " ");
      }
    }
  }),
  ...
});

server.run();
``` 

I run the app and there are no errors and the bug has disappeared.

Long story short, all documentation references to the Logger namespace are different than the actual implementation code.

Could this issue be notified of the change so myself and/or others can correct their logging instantiation?

**Comment by ebebbington** [comment]
I'll add onto this, you can see how to create a logger from which object by doing the following:

```
console.log(Drash.Loggers) /// {}
console.log(Drash.CoreLoggers) // { ConsoleLogger: [Function: ConsoleLogger], FileLogger: [Function: FileLogger], Logger: [Function: Logger] }
```

I now see that `Drash.Loggers` is used to **add a logger** , for example:
```
const logger = new Drash.CoreLoggers.ConsoleLogger({
  enabled: true,
  level: "all",
  tag_string: "{datetime} | {level} |",
  tag_string_fns: {
    datetime() {
      return new Date().toISOString().replace("T", " ");
    }
  }
});
Drash.addLogger('testLogger', logger)
console.log(Drash.Loggers)
// { testLogger: ConsoleLogger { test: false, configs: { enabled: true, level: "all", tag_string: "{datetime} | {level} |", tag_string_fns: { datetime: [Function: datetime] } } } }
```

And `Drash.CoreLoggers` is used for built-in loggers. The documentation doesn't actually explain this which leads to some confusion

**Comment by crookse** [comment]
@ebebbington ,

Sorry. There's code in master that changes the `ConsoleLogger`'s namespace to `Drash.CoreLoggers`. It hasn't been released yet, but you were able to catch it because the documentation's example code blocks use master.

I'm going to update the docs to reflect `CoreLoggers`. Sorry for the inconvenience again!
